### PR TITLE
[fix] packagejson update of missing module

### LINKED ISSUE #22: Add "supertest" module in package.json
Package.json missing two modules
- supertest
- pug

### PR DESCRIPTION
**Pull request by vineetvk01**
Closes #22 
1. Updated Package JSON with supertest
2. Removed pug and views.

### CODE REVIEW

File: server/src/app.js [hunk 1]
```diff
@@ -9,8 +9,8 @@ var indexRouter = require('./routes/index');
 var app = express();
 
 // view engine setup
-app.set('views', path.join(__dirname, 'views'));
-app.set('view engine', 'pug');
+// app.set('views', path.join(__dirname, 'views'));
```
Comment (nimish-gupta) [inline server/src/app.js]: Please remove comments.

File: server/src/app.js [hunk 2]
```diff
@@ -9,8 +9,8 @@ var indexRouter = require('./routes/index');
...
+// app.set('view engine', 'pug');
```
Comment (nimish-gupta) [inline server/src/app.js]: Why are you guys installing pug or setting up view engine?

**Review by nimish-gupta — DISMISSED** [review]

**Review by nitishmittal1990 — APPROVED** [review]

File: server/src/app.js (hunk 1 above)
Comment (vineetvk01) [reply server/src/app.js]: Yes removing, view was needed by @nitishmittal1990 for some testing with babel.

File: server/src/app.js (hunk 2 above)
Comment (vineetvk01) [reply server/src/app.js]: Have consulted team, removed the view engine. Please re-review.

**Review by fenilgandhi — APPROVED** [review]

**Pull request merged**